.github/workflows/main.yml

```
@@ -7,11 +7,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
         wagtail-version:
           - 4.1.5
           - 4.2.3
```

Could these be changed to
```suggestion
          - 4.1
          - 4.2
```

---

Removed patches:
https://github.com/torchbox-forks/wagtail-geo-widget/pull/1/commits/77d8821994d115628830f65c524924eb647f140d